### PR TITLE
Unified name

### PR DESCRIPTION
**Pull request by dcho5**
cachito_nexus_npm_proxy_repo_url is now cachito_nexus_npm_proxy_url

### CODE REVIEW

**Comment by chmeliik** [comment]
Travis CI is failing, looks like a few unit tests need fixing.

To make sure CI will pass, you can run unit tests locally with `tox` or `make test-unit`.

I think the name will also have to be changed in a few other places. Unit tests may not necessarily catch those because we use [mocking](https://docs.python.org/3/library/unittest.mock.html), so you might want to run integration tests as well - `tox -e integration` or `make test-integration`. These will only work if you have cachito running locally, see https://github.com/release-engineering/cachito#integration-tests

**Comment by tkdchen** [comment]
Do we expect the commits are singed-off?

File: cachito/workers/config.py [hunk 1]
```diff
@@ -270,14 +270,14 @@ def validate_npm_config():
...
+    if not conf.get("cachito_nexus_npm_proxy_url"):
         raise ConfigError(
-            'The configuration "cachito_nexus_npm_proxy_repo_url" must be set for this package '
+            'The configuration "cachito_nexus_npm_proxy_url" must be set for this package '
             "manager"
         )
 
 
-# yarn and npm use the same configuration options including `cachito_npm_proxy_repo_url`;
+# yarn and npm use the same configuration options including `cachito_npm_proxy_url`;
```
Comment (chmeliik) [inline cachito/workers/config.py:280]:
```suggestion
# yarn and npm use the same configuration options including `cachito_nexus_npm_proxy_url`;
```
This one was wrong before, might as well use this opportunity to fix it :-)

**Review by chmeliik — COMMENTED** [review]
Like in #473, please squash your commits into one and update your commit message with all the necessities, see https://github.com/release-engineering/cachito/pull/473#issuecomment-856537669

Apart from that, looks good!

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Comment by tkdchen** [comment]
:+1:

**Pull request merged**